go.mod

```diff
@@ -160,7 +160,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.59.1
 	github.com/DataDog/datadog-go/v5 v5.6.0
 	github.com/DataDog/datadog-operator v0.7.1-0.20241219210556-f517775059d1
-	github.com/DataDog/ebpf-manager v0.7.4
+	github.com/DataDog/ebpf-manager v0.7.6-0.20241223104659-fb57b17b70e0
```

Comment:
why do we use a commit from ebpf-manager?

Reply:
I can cut a new release and use that